src/com/estafet/learning/US008/Invoice.java

```diff
@@ -0,0 +1,10 @@
+package com.estafet.learning.US008;
+
+public abstract class Invoice extends Order {
```

Comment:
what purpose inheriting from `Order` class ?

Reply:
removed it.